src/core/variable.jl

```diff
@@ -127,6 +128,68 @@ function variable_reactive_line_flow(pm::GenericPowerModel; bounded = true)
     return q
 end
 
+############## DC Lines ############################################
+function variable_line_flow_dc(pm::GenericPowerModel; kwargs...)
+    variable_active_line_flow_dc(pm; kwargs...)
+    variable_reactive_line_flow_dc(pm; kwargs...)
+end
+
+"variable: `p_dc[l,i,j]` for `(l,i,j)` in `arcs_dc`"
+function variable_active_line_flow_dc(pm::GenericPowerModel; bounded = true)
+  pmin = Dict([(a, 0.0) for a in pm.ref[:arcs_dc]])
+  pref = Dict([(a, 0.0) for a in pm.ref[:arcs_dc]])
+  pmax = Dict([(a, 0.0) for a in pm.ref[:arcs_dc]])
+  loss0 = Dict([(a, 0.0) for a in pm.ref[:arcs_dc]])
+  br_status = Dict([(a, 0) for a in pm.ref[:arcs_dc]])
+  for (l,i,j) in pm.ref[:arcs_from_dc]
+      pmin[(l,i,j)] =  pm.ref[:dcline][l]["pminf"]
+      pmax[(l,i,j)] =  pm.ref[:dcline][l]["pmaxf"]
+      pmin[(l,j,i)] =  pm.ref[:dcline][l]["pmint"]
+      pmax[(l,j,i)] =  pm.ref[:dcline][l]["pmaxt"]
+      pref[(l,i,j)] =  pm.ref[:dcline][l]["pf"]
+      pref[(l,j,i)] =  pm.ref[:dcline][l]["pt"]
+      loss0[(l,i,j)] =  0
+      loss0[(l,j,i)] =  pm.ref[:dcline][l]["loss0"]
+      br_status[(l,i,j)] =  pm.ref[:dcline][l]["br_status"]
+      br_status[(l,j,i)] =  pm.ref[:dcline][l]["br_status"]
+  end

```

Comment:
See previous note regarding `br_status`, these should all be 1.0 by the time we get here.